phone_calls/java/PhoneCallsTest.java

```diff
@@ -94,10 +94,10 @@ public void testXMLMigration() throws FileNotFoundException, XMLStreamException
     public void testQueries() throws FileNotFoundException {
         List<Queries.QueryExample> queryExamples = Queries.getTestSubjects();
 
-        Queries.processSelection(0, queryExamples, databaseName);
-
         CSVMigration.main(new String[]{databaseName});
 
+        Queries.processSelection(0, queryExamples, databaseName);
+
```

Comment:
Ditto.

Reply:
Ditto to what, sorry? 😛

Reply:
I may have done these in a weird order. See nodejs/test.js:80-83.